build.yaml

```diff
@@ -31,14 +26,34 @@ steps:
        valueFrom: default_ns.name
    public: false
    secrets:
-    # hail|test-batch
-    - user-jwt-vkqfw
-    - gsa-key-2x975
-    # pipeline|test-user
-    - pipeline-test-0-1--hail-is-service-account-key
+    - test-gsa-key
    dependsOn:
     - default_ns
     - deploy_batch_sa
+ - kind: runImage
+   name: create_accounts
+   image: "{{ base_image.image }}"
+   script: |
+     set -e
+     echo creating jwt-signing-key secret...
+     python3 -c "
+     import secrets, json, jwt
```

Comment:
need to not do this in production

Reply:
I added a `scopes` field to step which is a list of scopes to run in.  There are currently two scopes, test and deploy.  If no scopes are specified, the step is always run.  I made this step scopes: ["test"].